tests/fe/crash_01.cc

```diff
@@ -46,7 +46,12 @@ test(FiniteElement<dim> &fe)
   AffineConstraints<double>    constraints;
   Functions::ZeroFunction<dim> boundary_values(fe.n_components());
   VectorTools::project_boundary_values_curl_conforming_l2(
-    dof_handler, 0, boundary_values, 0, constraints);
+    dof_handler,
+    0,
+    boundary_values,
+    0,
+    constraints,
+    StaticMappingQ1<dim>::mapping);
```

Comment:
I had a quick look at the declaration in `vector_tools.h`. Shouldn't it be possible to define two sets of functions, one with a `StaticMappingQ1` as default argument and one with a mapping collection without default argument? That should essentially be backwards compatible, shouldn't it?

Reply:
It is backwards compatible for the `DoFHandler`, but not for `hp::DoFHandler`, is is?

Reply:
The question is whether we need to end up in the other function with hp dof handler - my point is that the default mapping Q1 should be possible to use also with the hp dof handler. But it could turn out to be fuzzy. But then the question is whether many people actually used the hp facilities with curl/div conforming projections.

Reply:
I don't have a strong feeling about this. But, we discussed these days to eliminate mapping as default parameter, so this would fit to this intend!?